doc/sphinx/conf.py

```diff
@@ -220,7 +220,8 @@ def setup(app):
     'versions': [
         ("dev", "https://coq.github.io/doc/master/refman/"),
         ("stable", "https://coq.inria.fr/distrib/current/refman/"),
-        ("v8.14", "https://coq.github.io/doc/v8.14/refman/"),
+        ("8.15", "https://coq.github.io/doc/v8.15/refman/"),
+        ("8.14", "https://coq.github.io/doc/v8.14/refman/"),
```

Comment:
What we usually do at this point is to stabilize the URL of the previous release, though in this case, it would be `V8.14.1` but it hasn't been tagged yet. IINM it should happen pretty soon (when @silene is available again to take care of this) because all the 8.14.1-relevant PRs have been merged already, and the majority has even been backported already. So, we could wait for this before merging this PR, if that's fine for you, or we could come back and fix this later if you prefer.

Reply:
https://coq.inria.fr/distrib/V8.14.1/refman/ exists now, so I think the PR can be updated with this link instead.

Reply:
Oh actually, it looks like an 8.14.2 milestone exists already, so this might not be the last version of the 8.14 branch.

Reply:
This is kind of unrelated. There will be a 8.14.2 release only if a severe bug warrants it. The milestone is only there to keep track of things that would be part of 8.14.2 in that case. So, to the best of my current incomplete knowledge, 8.14.1 is the last one.

Reply:
Thanks for the precision. In this case, @SkySkimmer, I suggest you point to the fixed (8.14.1) refman version, and we'll update the link if ever an 8.14.2 release happened.